packages/site/cms-site/src/cookies/useLocalStorageCookieApi.ts

```diff
@@ -1,20 +1,31 @@
 import React from "react";
 import { useLocalStorage } from "usehooks-ts";
 
-import { CookieApiHook } from "./CookieApiHook";
+import { CookieApi } from "./CookieApi";
 
 const localStorageCookieApiKey = "comet-dev-cookie-api-consented-cookies";
 
+declare global {
+    interface Window {
+        cometLocalStorageCookieApi: ReturnType<CookieApi>;
+    }
+}
+
 /**
  * Only for use in development environment.
  */
-export const useDevCookieApi: CookieApiHook = (): ReturnType<CookieApiHook> => {
+export const useLocalStorageCookieApi: CookieApi = (): ReturnType<CookieApi> => {
     const [consentedCookies, setConsentedCookies] = useLocalStorage<string[]>(localStorageCookieApiKey, []);
+    const [simulateLoadingCookieProvider, setSimulateLoadingCookieProvider] = React.useState(true);
 
     React.useEffect(() => {
         const storedCookies = window.localStorage.getItem(localStorageCookieApiKey);
         const cookiesList = JSON.parse(storedCookies ?? "[]");
         logCookieUpdate(cookiesList);
+
+        setTimeout(() => {
+            setSimulateLoadingCookieProvider(false);
+        }, 1000);
```

Comment:
This timeout needs to be cleared in the useEffect cleanup function

Reply:
Done in 8088330cc921e1442ee85123338321c314e572e8